docs/faqs.md

```diff
@@ -97,11 +97,19 @@ No, recipes do not need to export a recipe license. Recipes and all files contri
 We generally consider tools like CMake as a standard tool to have installed in your system. Having the `cmake` package as a build require in **all** the recipes that use it will be an overkill, as every build requirement is installed like a requirement and takes time to download. However, `cmake` could still be useful to use in your profile:
 
 ```
-[build_requires]
+[tool_requires]
 cmake/3.17.2
 ```
 
-Other packages using more unusual build tools, like `OpenSSL` using `strawberryperl`, will have the build require in the recipe as it is likely that the user that want to build it from sources will not have it installed in their system
+Other packages using more unusual build tools, like `OpenSSL` using `strawberryperl`, will have the build require in the recipe as it is likely that the user that want to build it from sources will not have it installed in their system.
+
+## How are rare build systems without generators packaged?
+
+The C++ ecosystem has a lot of rare, unique and obscure build systems. Some of these are available in ConanCenter but they do not have built-in generators from the main Conan client.
+For these cases, contributors are asked to help reviewers as much as possible as it's likely we will not have expertise.
```

Comment:
I think something more actionable is needed, like "the recipe must explicitly encode the specifics of the build system, mapping the ``settings``, ``options`` for the binary configuration, and also mapping ``self.dependencies`` so the build system can locate the dependencies libraries``, or something in that line? Maybe some link to some example recipe?

Reply:
yes, I suppose we should illustrate with examples (links to existing recipes are okay) on how exactly do we:
- encode the specifics of the build system (I am personally not sure what does it mean, what specifics and how to encode?)
- mapping the settings, options for the binary configuration
- mapping self.dependencies so the build system can locate the dependencies libraries

in general, we should try our best to avoid vague/abstracts statements on **what** to do, instead explaining **how** to do things specifically. we cannot expect all our readers, especially newcomers, to know about all these things.

Reply:
Are there up to date docs for writing generators? https://docs.conan.io/en/latest/reference/generators/custom.html?highlight=generators https://docs.conan.io/en/latest/howtos/custom_generators.html#dyn-generators seem like the old stuff

https://docs.conan.io/en/2.0/search.html?q=generators&check_keywords=yes&area=default Nothing in the 2.0 docs

Reply:
```suggestion
The recipe is expected to encode the specifics of the build system, mapping the `settings`, `options` for the binary configuration, and also mapping `self.dependencies` so the build system can locate the dependencies libraries as required.
For these cases, contributors are asked to help reviewers as much as possible as it's likely we will not have expertise.

> TODO: Add a link to docs.conan.io which explains how to write a custom generator in the 2.0 sense
```